item_groups.json

```diff
@@ -0,0 +1,174 @@
+[
+{
+    "type" : "item_group",
+    "id" : "child_items",
+    "items":[
+	["b_dried_apple", 15]
+	]

```

Comment:
Please use linter (http://dev.narc.ro/cataclysm/format.html) to properly format your changes.

Reply:
Okay linted them up, didn't see that link earlier.